src/slides/slide-refs.yaml

```diff
@@ -0,0 +1,34 @@
+---
+slides:
+    - type: title
+      title: Slide References
+      subtitle: Use custom names to reference a slide
+      ref: title
+    - type: default
+      content: |
+        This slide will be sticky.
+        
+        Normally, you need to know the slide index ahead of time;
+        however, now you can specify a custom name (ref) for when
+        the slide will be sticky until.
+        
+        This will be sticky until the slide with ref `nail-polish-remover` is found.
+      sticky: nail-polish-remover
+    - type: default
+      content: |
+        A slide does not need to contain a ref.
+      
+        *Refs are available on slides through the `customRef` prop.*
+    - type: default
+      content: |
+        Refs are only a QoL feature for people making slides. They are precomputed
+        before rendering so that no duplicate refs exist and if in the future, the
+        need to return to a previous slide arises.
+        
+        An example might be to reference a presentation by slide with a URL.
+        `softe.club/slides/learn-git#making-a-pr` for example.
+    - type: default
+      content: I am the NAIL POLISH REMOVER SLIDE. NO MORE STICKY.
+      ref: nail-polish-remover
+    - type: default
+      content: And that's how simple slide refs are!
```

Comment:
This is documentation that needs to be added to `src/articles/docs/slidedecks.md`.